### PR TITLE
Canavandl/0.12.5 fixes

### PR DESCRIPTION
**Pull request by canavandl**
* Add summary
* Add deprecation note of --host parameter
* Unhide blog post

### CODE REVIEW

**Comment by canavandl** [comment]
ping @bryevdv 

We can merge this when we want to "go live"

File: content/blog/release-0-12-5/contents.lr [hunk 1]
```diff
@@ -2,10 +2,12 @@ title: Bokeh 0.12.5 Released
...
-Bokeh 0.12.5 has been released, bringing .
+Bokeh 0.12.5 has been released, bringing support for callbacks on user
+interaction events, interactive legends, custom computed transforms, and
+much more.
 
 ---
-pub_date: 2017-03-23
+pub_date: 2017-04-05
 ---
 author: Bryan Van de Ven
```
Comment (bryevdv) [inline content/blog/release-0-12-5/contents.lr]: I think multiple authors are supported or the one line could be "A and B"

File: content/blog/release-0-12-5/contents.lr [hunk 2]
```diff
@@ -216,6 +219,11 @@ custom events as well.
 
 #### Bokeh Server Options Simplified
 
+The ``--host`` parameter to the Bokeh Server CLI has been obviated and
+deprecated. It will be removed and trigger an error in a future release. If
+you're running a Bokeh Server process from a different address than your
+webserver, use the ``--allow-websockets-origin`` parameter.
+
```
Comment (bryevdv) [inline content/blog/release-0-12-5/contents.lr]: Suggest:

> The ``--host`` parameter to the Bokeh Server CLI was a common source of confusion. Recent changes to the Bokeh server have rendered it no longer necessary, so it is now deprecated. In order not to break existing deployments, the parameter is still accepted, but will be ignored. If you are embedding a Bokeh server application in a page that has a different public URL than the Bokeh server, you will want to supply the ``--allow-websockets-origin`` parameter to allow connections originating from the public URL.

File: content/blog/release-0-12-5/contents.lr (hunk 1 above)
Comment (canavandl) [inline content/blog/release-0-12-5/contents.lr]: I didn't want to add two titles and two twitter handles, it made the top look weird. I think I'll leave my name off this on and write the next one.

**Pull request merged**